### ISSUE TITLE
EuropeanOpen && EEGPSR in branch Develop.

### ISSUE THREAD

**Issue by kyordhel** [kyordhel/GPSRCmdGen2015#8]
To those who are following this repo.

All changes and updates for 2016 (specially the European Open and the EEGPSR command generation)  are now on Branch Develop. Please pay spetial attention to Category 3 which has major changes and challeges.

I'm looking forward to read your feedback in order to fix and merge by next saturday.

**Comment by LoyVanBeek** [comment]
Looks OK to me. 
Category 3 sometimes yields nonsensical responses like below, but in that case I'll skip and generate a new one. 

How do I generate EE-GPSR commands?

```
grasp the banana to milk

banana
     replace with milk at the bin
milk
     replace with John in the bathroom

======================================================================================================================================================================================================================================

Press q to quit, c to clear.
Enter category 1, 2, or 3: 3
Choosen category 3
Selected Incongruent Commands (High difficulty degree grammar).

======================================================================================================================================================================================================================================

get the beer to chocolate

beer
     replace with chocolate at the bathroom locker
chocolate
     replace with me
```

**Comment by kyordhel** [comment]
For GPSR run
GPSRCmdGen/bin/Release/GPSRCmdGen.exe

For EEGPSR run
EEGPSRCmdGen/bin/Release/EEGPSRCmdGen.exe

Will check incongruent grammar, it should be easy to read. However, even if the output is nonsensical, this behaviour is intended. Just delete the Incongruent grammar file and the generator will generate such cases no more.

**Comment by kyordhel** [comment]
Solved in c84c2a1
